### PR TITLE
adding guides section in documentation

### PR DESCRIPTION
**Pull request by peyanski**

### CODE REVIEW

File: docusaurus.config.js [hunk 1]
```diff
@@ -24,9 +24,10 @@ module.exports = {
       title: 'HACS',
       links: [
         { to: 'docs/installation/prerequisites', label: 'Installation', position: 'left' },
-        { to: 'docs/configuration/start', label: 'Configuration', position: 'left' },
+	{ to: 'docs/configuration/start', label: 'Configuration', position: 'left' },
         { to: 'docs/basic/getting_started', label: 'Usage', position: 'left' },
-        { to: 'docs/default_repositories', label: 'Default repositories', position: 'left' },
+        { to: 'docs/guides/guides', label: 'Guides', position: 'left' },
```
Comment (ludeeus) [inline docusaurus.config.js]:
```suggestion
        { to: 'docs/guides/guides', label: 'Community guides', position: 'left' },
```

File: docusaurus.config.js [hunk 2]
```diff
@@ -24,9 +24,10 @@ module.exports = {
       title: 'HACS',
       links: [
         { to: 'docs/installation/prerequisites', label: 'Installation', position: 'left' },
-        { to: 'docs/configuration/start', label: 'Configuration', position: 'left' },
+	{ to: 'docs/configuration/start', label: 'Configuration', position: 'left' },
```
Comment (ludeeus) [inline docusaurus.config.js]:
```suggestion
	  { to: 'docs/configuration/start', label: 'Configuration', position: 'left' },
```

File: sidebars.js [hunk 3]
```diff
@@ -34,6 +34,11 @@ module.exports = {
       "installation/remove"
     ]
   },
+  "guides": {
```
Comment (ludeeus) [inline sidebars.js]: This section does not need a sidebar

File: documentation/guides/guides.md [hunk 4]
```diff
@@ -0,0 +1,12 @@
+---
+id: guides
+title: Guides
+description: "Guides, Tutorials &  Videos about HACS"
+---
+
+## Videos
+
+**Here you can find Guides, Tutorials & Videos about how  to install/use HACS.**
```
Comment (ludeeus) [inline documentation/guides/guides.md]:
```suggestion
_Here you can find community made guides, tutorials & videos about how  to install/use HACS._

**NB!: Most of what you find here is static, and may not be up to date.**
```

File: documentation/guides/guides.md [hunk 5]
```diff
@@ -0,0 +1,12 @@
+---
+id: guides
+title: Guides
```
Comment (ludeeus) [inline documentation/guides/guides.md]:
```suggestion
title: Community guides
```

File: documentation/guides/guides.md [hunk 6]
```diff
@@ -0,0 +1,12 @@
...
+description: "Guides, Tutorials &  Videos about HACS"
+---
+
+## Videos
+
+**Here you can find Guides, Tutorials & Videos about how  to install/use HACS.**
+
+- Home Assistant Themes and Plugins with HACS ![image](/img/hacs-video1.png)
```
Comment (ludeeus) [inline documentation/guides/guides.md]:
```suggestion
### Home Assistant Themes and Plugins with HACS! (by [KPeyanski](https://www.youtube.com/channel/UCiyU6otsAn6v2NbbtM85npg))

[![yt](https://img.youtube.com/vi/4cOdgW23KCA/0.jpg)](https://www.youtube.com/watch?v=4cOdgW23KCA"Home Assistant Themes and Plugins with HACS!")
```

File: documentation/guides/guides.md [hunk 7]
```diff
@@ -0,0 +1,12 @@
...
+title: Guides
+description: "Guides, Tutorials &  Videos about HACS"
+---
+
+## Videos
+
+**Here you can find Guides, Tutorials & Videos about how  to install/use HACS.**
+
+- Home Assistant Themes and Plugins with HACS ![image](/img/hacs-video1.png)   
+https://youtu.be/4cOdgW23KCA
```
Comment (ludeeus) [inline documentation/guides/guides.md]:
```suggestion

```

File: sidebars.js (hunk 3 above)
Comment (peyanski) [reply sidebars.js]: removed

**Comment by ludeeus** [comment]
ill fix that in master

**Comment by peyanski** [comment]
Thanks @ludeeus It was a real pleasure :)

**Pull request merged**